internal/namespace/manager.go

```diff
@@ -850,6 +880,16 @@ func (nm *namespaceManager) loadNamespace(ctx context.Context, name string, inde
 		}
 	}
 
+	//
+	// Handle TLS Configs
+	tlsConfigArray := conf.SubArray(coreconfig.NamespaceTLSConfigs)
+	tlsConfigs := make(map[string]*tls.Config)
+
+	err = nm.loadTLSConfig(ctx, tlsConfigs, tlsConfigArray)
```

Comment:
Think it's great this is happening here. I believe (and would like to validate) this means:
- This happens automatically on each dynamic reload
- If we fail because we have bad TLS config for this namespace, then **only** this namespace will fail to init

Reply:
Correct - will add a test for the failure case

Reply:
Ok - I really want to be engaged in the discussion if/when we make a change to ^^^ - as I think it's subtle where the boundary is between bug and intentional safe behavior... and I've been iterating a lot in that area of dynamic reload on that.

Let's merge with the behavior as it is, and treat any change as a separate PR.